### DISCUSSION TITLE
free wifi anywhere you go!

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#519] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
if you go to cms schools *cough 3kho cough* then heres the wifi password
Name: CMSsMHH
Identity: cmsgoogle
Password: CM$G00gl3wire<

ok hahahah byee!!!!

**Comment by obkduishinijf** [comment]
THAT IS SO COOL 🔥 🔥 🔥 🔥 🔥 YOU GET 5 FIRE EMOJIS
